client/status.v2/status.go

```diff
@@ -17,14 +17,18 @@ type Client interface {
 	Status() (daemon.Status, error)
 	StatusRaw() ([]byte, error)
 	SignalWakeup(job string) error
+	SignalSnapshot(job string) error
 	SignalReset(job string) error
 }
 
-var statusv2Flags struct {
-	Mode choices.Choices
-	Job  string
+type statusFlags struct {
+	Mode  choices.Choices
+	Job   string
+	Delay uint
```

Comment:
I didn’t think about using time.Duration directly, I just changed this and now to set the delay via cli you need to specify a time, like –d 3s, not sure whether this is the best way. I also set the minimum delay to 1 sec to avoid high cpu usage for example when using –d 1ms or even less as zrepl UI would kill the cpu.